File: Source_code_BUIDUCLONG/C#_OOP/LoaiThietBi.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CK_OOP_BuiDucLong
{
    internal class LoaiThietBi
    {
        public string maLoai { get; set; } // Mã loại
        public string theLoai { get; set; } // Thể loại
        public string donViTinh { get; set; } // Đơn vị tính
        public string ghiChu { get; set; } // Ghi chú

        public LoaiThietBi(string maLoai, string theLoai, string donViTinh, string ghiChu, ICollection<ThietBi> dsThietBi)
        {
            this.maLoai = maLoai;
            this.theLoai = theLoai;
            this.donViTinh = donViTinh;
            this.ghiChu = ghiChu;
            this.dsThietBi = dsThietBi;
        }
        public LoaiThietBi(LoaiThietBi ltb)
        {
            maLoai = ltb.maLoai;
            theLoai = ltb.theLoai;
            donViTinh = ltb.donViTinh;
            ghiChu = ltb.ghiChu;
            dsThietBi = ltb.dsThietBi;
        }
        public ICollection<ThietBi> dsThietBi { get; set; }
    }
}

File: Source_code_BUIDUCLONG/Java_OOP/src/mypack/LoaiThietBi.java
package mypack;

import java.util.List;

public class LoaiThietBi {
	public String maLoai; // Mã loại
    public String theLoai; // Thể loại
    public String donViTinh; // Đơn vị tính
    public String ghiChu; // Ghi chú

    public LoaiThietBi(String maLoai, String theLoai, String donViTinh, String ghiChu, List<ThietBi> dsThietBi)
    {
        this.maLoai = maLoai;
        this.theLoai = theLoai;
        this.donViTinh = donViTinh;
        this.ghiChu = ghiChu;
        this.dsThietBi = dsThietBi;
    }
    public LoaiThietBi(LoaiThietBi ltb)
    {
        maLoai = ltb.maLoai;
        theLoai = ltb.theLoai;
        donViTinh = ltb.donViTinh;
        ghiChu = ltb.ghiChu;
        dsThietBi = ltb.dsThietBi;
    }
    public List<ThietBi> dsThietBi;
    
    public String getMaLoai()
    {
    	return maLoai;
    }
    public void setMaLoai (String maLoai)
    {
    	this.maLoai = maLoai;
    }
    public String getTheLoai()
    {
    	return theLoai;
    }
    public void setTheLoai (String theLoai)
    {
    	this.theLoai = theLoai;
    }
    public String getDonViTinh()
    {
    	return donViTinh;
    }
    public void setDonViTinh(String donViTinh)
    {
    	this.donViTinh = donViTinh;
    }
    public String getGhiChu()
    {
    	return ghiChu;
    }
    public void setGhiChu(String ghiChu)
    {
    	this.ghiChu = ghiChu;
    }
    public List<ThietBi> getDsThietBi()
    {
    	return dsThietBi;
    }
    public void setDsThietBi(List<ThietBi> dsThietBi)
    {
    	this.dsThietBi = dsThietBi;
    }
}
